source/features/unlink-image-auto-links.tsx

```diff
@@ -0,0 +1,46 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import GitHubURL from '../github-helpers/github-url';
+import {isMarkdownFile} from '../github-helpers';
+
+function isMarkupFile(): boolean {
+	if (isMarkdownFile()) {
+		return true;
+	}
+
+	// https://github.com/github/markup/blob/46908504f5099b15a88030749f0aeb52cd2f6799/README.md#markups
+	if (/\.(textile|rdoc|org|creole|(media)?wiki|rst|a(scii)?doc|asc|pod)$/.test(location.pathname)) {
```

Comment:
Let's inline all the related extensions here, without using external "markdown" checks